client/src/App.jsx

```diff
@@ -199,17 +200,17 @@ class App extends React.Component {
       ? (
         this.handleInitialStartup()
       )
-      : (
+      : window.innerWidth >= 1200 ? (
```

Comment:
@vangny @nathanong12 @N-Turner Imo nesting all of these ternaries is confusing.

I might try something like the following instead:
```
if (!isLoggedIn || !isLoaded) {
  return this.handleInitialStartup();
}
if (window.innerWidth >= 1200) {
  return (
    ...
  );
}
return (
  ...
);
```